Automattic/pocket-casts-ios
# End of Year: show top 1 podcast and top 5 podcasts

| 📘 Project: #376 |
|:---:|

Adds the story for the top 1 podcast and the top 5 podcasts. Designs are not contemplated in this task.

| Top 1 podcast | Top 5 podcasts |
| -------------- | --------------- |
| ![Simulator Screen Shot - iPhone 14 - 2022-10-19 at 17 16 49](https://user-images.githubusercontent.com/7040243/196795411-8dc3229a-094b-4626-b986-448d744935a1.png) | ![Simulator Screen Shot - iPhone 14 - 2022-10-19 at 17 16 51](https://user-images.githubusercontent.com/7040243/196795439-cb3ddc8e-4c1f-4466-9225-8e53c63a006a.png) |

## To test

1. Enable the `endOfYear` flag in `FeatureFlag.swift`
2. Run the app and make sure you're logged in to an account with a few items on Listening History
3. When the prompt appears, tap "View My 2022"
4. Skip to the 5th story
5. ✅ Check that it shows your most listened podcast, together with the number of episodes and listening time
6. Skip to the 6th story
7. ✅ Check that it shows your top 5 podcasts

## Checklist
